test/__snapshots__/index.spec.js.snap

```diff
@@ -43,9 +43,17 @@ Object {
       },
     ],
     Array [
-      "<PROJECT_ROOT>/node_modules/babel-plugin-lodash/lib/index.js",
+      Array [
+        "<PROJECT_ROOT>/node_modules/babel-plugin-lodash/lib/index.js",
+      ],
       Object {
-        "id": "lodash-compat",
+        "id": Array [
+          "lodash",
+          "lodash-es",
+          "lodash-compat",
+          "lodash/fp",
+          "lodash-compat",
```

Comment:
There's a repeated `lodash-compat` here. Can you change the test to use `recompose` instead?